### ISSUE TITLE
No `dist` directory after v3.7.1

### ISSUE THREAD

**Issue by scheakur** [ritz078/embed.js#114]
There is no directory `dist` when installing `embed-js` whose version is higher than or equal to 3.7.1.

```
$ npm install --save embed-js@3.7.0
$ ls node_modules/embed-js 
README.md         RELEASES.md       assets/           bower_components/ dist/             package.json      src/
```

```
$ npm install --save embed-js@3.7.1
$ ls node_modules/embed-js 
CHANGELOG.md      README.md         assets/           bower_components/ package.json      src/
```

But the `main` property in package.json specifies `./dist/embed.min.js`.

**Comment by ritz078** [comment]
fixed in v3.7.4

**Comment by scheakur** [comment]
Thank you :smile:
